src/core_codemods/https_connection.py

```diff
@@ -22,12 +22,24 @@
 
 class HTTPSConnection(BaseCodemod, Codemod):
     METADATA = CodemodMetadata(
-        DESCRIPTION=("Enforce HTTPS connection"),
+        DESCRIPTION="Enforce HTTPS Connection for urllib3",
```

Comment:
We need to fix the API for this, but in general `DESCRIPTION` should not be capitalized because it is getting used as `CHANGE_DESCRIPTION` which is rendered by the platform as a comment on the PR.

Reply:
Aha gotcha, now I understand, will fix.